scripts/superglue-baselines.sh

```diff
@@ -30,7 +30,7 @@ function record() {
 }
 
 function rte() {
```

Comment:
Could you also add `winogender-diagnostic` to the other `rte` commands? (`rte_plus` and `rte_bow`)